pkg/rc/replication_controller.go

```diff
@@ -785,8 +786,31 @@ func (rc *replicationController) updateAllocations(ineligible []types.NodeName)
 		},
 	}
 
+	rc.mu.Lock()
+	auditLogDetails, err := audit.NewNodeTransferStartDetails(
+		rc.ID(),
+		rc.NodeSelector,
+		oldNode,
+		newNode,
+		rc.ReplicasDesired,
+	)
+	rc.mu.Unlock()
+	if err != nil {
+		return "", "", util.Errorf("could not generate node transfer start audit log details: %s", err)
+	}
+
 	writeCtx, writeCancel := transaction.New(context.Background())
 	defer writeCancel()
+
+	err = rc.auditLogStore.Create(
+		writeCtx,
```

Comment:
The error message in `util.Errorf("Could not commit CASTxn: %s", err)` on line 821 (I couldn't figure out how to add a comment to a line you didn't change) could be updated since the transaction is no longer just the CASTxn